into/backends/tests/test_iopro.py

```diff
@@ -0,0 +1,130 @@
+import sqlalchemy as sa
+from into import into, resource, convert
+from into.core import path
+from into.backends.sql import dshape_to_table
+from into.backends.iopro import (iopro_sqltable_to_ndarray,
+                                 iopro_sqlselect_to_ndarray)
+import numpy as np
+from numpy import ndarray, random
+from datashape import discover, dshape
+
+import iopro, iopro.pyodbc
+
+mysql_url = 'mysql+pymysql://continuum:continuum_test@localhost/continuum'
+pyodbc_mysql_url = 'DRIVER={mysql};SERVER=localhost;DATABASE=continuum;'\
+                   'UID=continuum;PASSWORD=continuum_test;OPTION=3;'
+
+#WARNING: pyodbc has no problem connecting to postgres,
+#  but sqlalchemy doesn't have a dialect to do so.
+#  So, we can't actually use a postgres database in this "convert" testing
+postgres_url = 'postgresql://continuum:continuum_test@localhost/continuum'
+pyodbc_postgres_url = 'DRIVER={postgresql};SERVER=localhost;DATABASE=continuum;'\
+                      'UID=continuum;PASSWORD=continuum_test;PORT=;OPTION=;'
```

Comment:
We should probably redirect these connection strings to match those on travis.ci.  See the `test_postgres_into.py` and `test_mysql_into.py` files for connection details.